lib/cli/src/frameworks/web-components/Button.stories.ts

```diff
@@ -1,14 +1,15 @@
-import { Button } from './Button';
+import { Story, Meta } from '@storybook/web-components';
```

Comment:
`frameworks/web-components` JS files should remain untouched as people are not forced to use typescript.